Assets/Packages/com.studio23.ss2.savesystem/Samples/Demo/Scripts/ItemDataBehaviour.cs

```diff
@@ -0,0 +1,25 @@
+using Studio23.SS2.SaveSystem.Interfaces;
+using UnityEngine;
+
+
+public class ItemDataBehaviour : MonoBehaviour, ISaveable
+{
+
+    public string _uniqueID;
```

Comment:
there should be instructions as [Title], for creating uniqueID,

Reply:
it's from sample.